module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -127,7 +130,8 @@ protected function translateList($transPrefix, $list)
     {
         $transList = [];
         foreach ($list as $current) {
-            $transList[] = $this->translateWithPrefix($transPrefix, $current);
+            // $current can be an array if pickValues() is called with callnumbers and a translation prefix
```

Comment:
Typo: wrong method name.

```suggestion
            // $current can be an array if pickValue() is called with callnumbers and a translation prefix
```